src/compiler/cache.ts

```diff
@@ -158,15 +158,12 @@ export class Cache implements d.Cache {
     }
   }
 
-  private getCacheFilePath(key: string) {
+  private getCacheFilePath(key: string): string {
     return join(this.config.cacheDir, key) + '.log';
   }
 
-  getMemoryStats() {
-    if (this.cacheFs != null) {
```

Comment:
There is a number of checks around `cacheFs` being defined. Is it safe to assume it is always defined (meaning we can simplify the other checks) or is it always guaranteed to be defined in the execution when this method would be called?

Reply:
I believe it is guaranteed to be defined, I can't find anywhere where it's reassigned after the initial assignment in the constructor so I think the checks are unnecessary -- I'm going to try removing them and see if anything breaks!

Reply:
nvm did some testing and we need those checks!

Reply:
Does anything need to get pushed back up to GH/changed here if we need these checks?

Reply:
yup, forgot to push reverting that! should be good now